gadget-container/gadgets/bcck8s/bcc-wrapper.sh

```diff
@@ -136,7 +136,31 @@ if [ "$MANAGER" = "true" ] ; then
     MODE="--cgroupmap"
     MAPPATH=$BPFDIR/gadget/cgroupidset_$TRACERID
   fi
-  exec $GADGET $MODE $MAPPATH "$@"
+  # Deal specifically with mountsnoop arguments.
+  if [ $(basename $GADGET) = 'mountsnoop' ]; then
+    argv=($@)
+
+    # mountsnoop does not support --containersmap.
+    # So, we need to remove --containersmap and its associated value.
+    for i in ${!argv[@]}; do
+      if [[ ${argv[i]} = '--containersmap' ]]; then
+        # Remove --containersmap
+        unset 'argv[i]'
+
+        # And its value.
+        i=$((i + 1))
+        unset 'argv[i]'
+
+        break
+      fi
+    done
+
+    # Call mountsnoop
+    exec $GADGET $MODE $MAPPATH "${argv[@]}"
+  else
+    # Call other managed gadget.
+    exec $GADGET $MODE $MAPPATH "$@"
+  fi
```

Comment:
The `--containersmap` argument in an addition that we maintain in our fork (https://github.com/kinvolk/bcc). I think after https://github.com/iovisor/bcc/pull/3596 is merged, we could update our fork to include it and we can add the `--containersmap` parameter to the tool. (If we don't want to support it we can just add the parameter to avoid the error and also to avoid having this gadget-specific logic here to remove that parameter).